### PR TITLE
Application Theming & Typographie enhancement.

### PR DESCRIPTION
**Pull request by ssakone**

### CODE REVIEW

**Comment by ssakone** [comment]
not ready.

**Comment by smk762** [comment]
Not sure what the purpose of the arrows on these buttons are
image

This looks a bit strange (default blue dark them in binance setting theme)
image

Buttons and tab highlight using default blue dark theme (should be yellow for binance theme?)
image


No mouseover effect?

https://user-images.githubusercontent.com/35845239/126266773-531852de-05df-40fd-bab6-65fa4e3978be.mp4

Button to enable more coins on wallet page is a bit off center.
image


Scrollbar is default blue
image
image

**Comment by ssakone** [comment]
image
@Smk i update the addressBook ui, what you think?

**Comment by smk762** [comment]
Looks better, thanks

**Comment by tonymorony** [comment]
@ssakone there is a conflict with dev atm:
`atomic_defi_design/qml/Screens/FirstLaunch.qml`, that's why GHA CI not triggered I think

**Comment by ssakone** [comment]
@tonymorony  @smk762 

Peek 2021-07-22 11-23
vs
Peek 2021-07-22 11-21

**Comment by smk762** [comment]
I prefer the second one.

**Comment by ssakone** [comment]
> I prefer the second one.

ok

**Comment by smk762** [comment]
maybe inverted tho?
Like solid for selected outline for hover

**Comment by ssakone** [comment]
> maybe inverted tho?
> Like solid for selected outline for hover

possible

**Comment by tonymorony** [comment]
as I think just border highlighting might be enough

File: atomic_defi_design/qml/Components/DexAppButton.qml [hunk 1]
```diff
@@ -1,21 +1,27 @@
...
+    property alias font: _label.font
     property alias leftPadding: _contentRow.leftPadding
     property alias rightPadding: _contentRow.rightPadding
     property alias topPadding: _contentRow.topPadding
     property alias bottomPadding: _contentRow.bottomPadding
-    property alias label: _label
+
+    property string text: ""
+    property string iconSource: ""
+    property string backgroundColor: enabled? _controlMouseArea.containsMouse? DexTheme.buttonColorHovered : DexTheme.buttonColorEnabled : DexTheme.buttonColorDisabled
```
Comment (SylEze) [inline atomic_defi_design/qml/Components/DexAppButton.qml:23]: Spaces

File: atomic_defi_design/qml/Exchange/Trade/SimpleView/SubBestOrder.qml [hunk 2]
```diff
@@ -157,13 +159,13 @@ DefaultListView
             DefaultText                         // Order Fiat Volume
             {
                 Layout.preferredWidth: _fiatVolumeColumnSize
-                text: price_fiat+API.app.settings_pg.current_fiat_sign
+                text: price_fiat+Constants.API.app.settings_pg.current_fiat_sign
```
Comment (SylEze) [inline atomic_defi_design/qml/Exchange/Trade/SimpleView/SubBestOrder.qml:162]: Spaces

File: atomic_defi_design/qml/Exchange/Trade/SimpleView/SubBestOrder.qml [hunk 3]
```diff
@@ -157,13 +159,13 @@ DefaultListView
...
             }
             
             DefaultText
             {
                 Layout.preferredWidth: _cexRateColumnSize
-                color: cex_rates=== "0" ? Qt.darker(theme.foregroundColor) : parseFloat(cex_rates)>0? theme.redColor : theme.greenColor
+                color: cex_rates=== "0" ? Qt.darker(DexTheme.foregroundColor) : parseFloat(cex_rates)>0? DexTheme.redColor : DexTheme.greenColor
```
Comment (SylEze) [inline atomic_defi_design/qml/Exchange/Trade/SimpleView/SubBestOrder.qml:168]: Spaces

**Comment by ssakone** [comment]
@SylEze  don't worry for operand thats old file, i will correct all later with js-beautify

**Comment by ssakone** [comment]
@smk762 @tonymorony @Milerius need review :)

**Comment by smk762** [comment]
On linux:
https://user-images.githubusercontent.com/35845239/126983746-0348c9df-aa20-40f0-9ab2-d8e4773d036f.mp4
Seen on other themes too.

- The only button I don't see a hover effect on is the "Apply filter" button in simple view.
- no overflow seen in English so far (other languages pending)

**Comment by smk762** [comment]
When clicking logout button in settings, it does not log out
```
[20:13:33] [warning] [main.prerequisites.hpp:92] [575484]: qrc:/atomic_defi_design/qml/Settings/SettingModal.qml:25: TypeError: Cannot read property 'accentColor' of undefined
```

**Comment by smk762** [comment]
login error resolved :heavy_check_mark: 
simple mode "apply filter" button hover effect ok :heavy_check_mark: 

Overflows detected :alarm_clock:  Mostly in settings pages.
image
image
image
image
image
image
image
image
image
image
image

**Comment by smk762** [comment]
Settings buttons fixed :heavy_check_mark: 
Notifications overflow resolved :no_entry: 
FAQ overflow resolved :no_entry:

**Comment by smk762** [comment]
Notification overflow :heavy_check_mark: 
FAQ overflow :heavy_check_mark: 


Portfolio overflow  :-1: 
image

**Comment by tonymorony** [comment]
I can't switch back to dark mode by the topbar icon after I switched to light

The "Add asset" button is almost not visible/noticeable, please add some contrast to it (via background or text)
image

**Comment by smk762** [comment]
> I can't switch back to dark mode by the topbar icon after I switched to light
> 
> The "Add asset" button is almost not visible/noticeable, please add some contrast to it (via background or text)
> image

```[18:31:44] [warning] [main.prerequisites.hpp:92] [59607]: QString::arg: Argument missing: END APPLY , Light
[18:31:44] [debug] [main.prerequisites.hpp:86] [59607]: END APPLY 
```

**Comment by ssakone** [comment]
> > I can't switch back to dark mode by the topbar icon after I switched to light
> > The "Add asset" button is almost not visible/noticeable, please add some contrast to it (via background or text)
> > image
> 
> ```
> [18:31:44] [warning] [main.prerequisites.hpp:92] [59607]: QString::arg: Argument missing: END APPLY , Light
> [18:31:44] [debug] [main.prerequisites.hpp:86] [59607]: END APPLY 
> ```

not sure to find this warning yet

**Comment by tonymorony** [comment]
still not switching back to the dark on my side

**Comment by ssakone** [comment]
> still not switching back to the dark on my side

can you show log?

**Comment by tonymorony** [comment]
> > still not switching back to the dark on my side
> 
> can you show me log?

https://paste.ubuntu.com/p/GWKN3h4bwv/

**Comment by ssakone** [comment]
> > > still not switching back to the dark on my side
> > 
> > 
> > can you show me log?
> 
> https://paste.ubuntu.com/p/GWKN3h4bwv/

the log show light theme applied, i don't understand why Ui not change,
@smk762  have you same issue in other platform?

**Comment by tonymorony** [comment]
> > > > still not switching back to the dark on my side
> > > 
> > > 
> > > can you show me log?
> > 
> > 
> > https://paste.ubuntu.com/p/GWKN3h4bwv/
> 
> the log show light theme applied, i don't understand why Ui not change,
> @smk762 have you same issue in other platform?

at first light theme applied, and then when theme is already light according to log it changing it to light again instead of dark

**Comment by ssakone** [comment]
dddd

**Comment by ssakone** [comment]
> > > > > still not switching back to the dark on my side
> > > > 
> > > > 
> > > > can you show me log?
> > > 
> > > 
> > > https://paste.ubuntu.com/p/GWKN3h4bwv/
> > 
> > 
> > the log show light theme applied, i don't understand why Ui not change,
> > @smk762 have you same issue in other platform?
> 
> at first light theme applied, and then when theme is already light according to log it changing it to light again instead of dark
> Screen.Recording.2021-08-03.at.04.49.20.mov

Your icon still not change, its the reason app think its always dark
strange

can you check manually in atomic_qt/theme/light.json if "theme" is in array?

**Comment by ssakone** [comment]
maybe new theme update does not charge in your computer

**Comment by tonymorony** [comment]
I see, I've had to "reset asset configuration" and after it, the switcher started to work fine

can we do this reset automagically when it's needed? maybe we should add precise versioning to this configuration hmm, otherwise if the user catch something like that he'll think that it's the bug

**Comment by ssakone** [comment]
> I see, I've had to "reset asset configuration" and after it, the switcher started to work fine
> 
> can we do this reset automagically when it's needed? maybe we should add precise versioning to this configuration hmm, otherwise if the user catch something like that he'll think that it's the bug

yes maybe, i need to check this with @Milerius

**Comment by tonymorony** [comment]
noticed some elements contrast lost on dark theme: 

Screenshot 2021-08-06 at 13 48 00
Screenshot 2021-08-06 at 13 47 35

**Comment by ssakone** [comment]
> noticed some elements contrast lost on dark theme:
> 
> Screenshot 2021-08-06 at 13 48 00
> 
> Screenshot 2021-08-06 at 13 47 35
> 
> can see you worked on login page, please note that logo text on this page is not contrast still:
> Screenshot 2021-08-06 at 13 50 26
> 
> Screenshot 2021-08-06 at 14 21 51
> 
> Screenshot 2021-08-06 at 14 23 09
> 
> Screenshot 2021-08-06 at 14 23 44

Button and logo also fixed, i belive wrong json, i doest not push with it, roman will sync it later

**Comment by ssakone** [comment]
image

**Comment by tonymorony** [comment]
arrows not visible here now btw:
Screenshot 2021-08-06 at 17 59 34

**Comment by tonymorony** [comment]
app is totally broken for me on the last build [ @ssakone @Milerius ]
image

**Comment by Milerius** [comment]
It's was my fault - should be fixed now, need testing and we need to check if this work on osx

**Review by tonymorony — APPROVED** [review]
Let's merge this one since it's improving a lot of things with an aim of light theme improving in near future

**Comment by smk762** [comment]
Mac was looking ok here on latest build

**Review by smk762 — APPROVED** [review]
Any further tweaks to lite mode would be more cleanly done in another PR

**Pull request merged**